colorsys.js

```diff
@@ -344,9 +343,17 @@ colorsys.decimal_to_hex = colorsys.decimalToHex = colorsys.decimal2Hex
 
 colorsys.decimal2Hex = function(decimalColor) {
   if (typeof decimalColor === "string") {
-    return "#" + parseInt(decimalColor).toString(16)
+    let hex = parseInt(decimalColor).toString(16)
```

Comment:
Here it is an implementation without typechecks https://github.com/netbeast/colorsys/pull/25/files?w=1

If the developer uses the library wrongly the objective is this library is to be transparent. Not to throw unless the developer causes a throw, and if the input is wrong, show a wrong output.

Feel free to merge my PR over this, or close it and merge this one.

PS: Use single quotes `'`